tidy3d/web/api/webapi.py

```diff
@@ -48,8 +48,10 @@ def run(
     solver_version: str = None,
     worker_group: str = None,
 ) -> SimulationDataType:
-    """Submits a simulation to the server, starts running, monitors progress,
-    downloads, and loads results as a corresponding data object.
+    """
+
```

Comment:
rm newline?

Reply:
Sorted. I'll also check all the content post the webapi restructure.